app/src/controllers/organizations/organization/selectors.js

```diff
@@ -24,3 +30,20 @@ export const activeOrganizationLoadingSelector = (state) =>
   organizationSelector(state).organizationLoading || false;
 
 export const activeOrganizationNameSelector = (state) => activeOrganizationSelector(state)?.name;
+
+export const activeOrganizationRolesSelector = createSelector(
```

Comment:
Looks like this selector too similar with `userRolesSelector`.
Can we leave just one?
Do we really need to select user roles from different sources?
Can we just extract the slugs from the URL and define org/project user role based on `assignedOrganizations` and `assignedProjects`. For some cases they may be empty (for admin/manager viewing the project/org without assignment).

Reply:
Done.